docs/src/pages/components/textarea-autosize/MaxHeightTextarea.tsx

```diff
@@ -7,8 +7,7 @@ export default function MaxHeightTextarea() {
       rowsMax={4}
       aria-label="maximum height"
       placeholder="Maximum 4 rows"
-      defaultValue="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt
-          ut labore et dolore magna aliqua."
+      defaultValue={'Lorem ipsum dolor sit amet. '.repeat(4)}
```

Comment:
I'm sorry, I shouldn't have proposed repeat, it's not supported by IE 11.

Reply:
In `ScrollDialog.js`, we use the following, It might be overkill:
```js
[...new Array(50).map().joint('').
```

I guess there isn't much opportunity for simplification, It might be a dead end 🤔.